sqlmesh/dbt/profile.py

```diff
@@ -26,19 +26,27 @@ def __init__(
         path: Path,
         target_name: str,
         target: TargetConfig,
+        project_model_config: t.Dict[str, t.Any],
```

Comment:
This doesn't look right to me. IMHO the `profile` object should not be aware of the project config.

Reply:
What's the idea behind putting it here?